server/cmwell-ws/app/controllers/Application.scala

```diff
@@ -442,18 +442,19 @@ callback=< [URL] >
       case Failure(e) => asyncErrorHandler(e)
       case Success(request) => {
         val normalizedPath = normalizePath(request.path)
+        val isPriorityWrite = originalRequest.getQueryString("priority").isDefined
```

Comment:
Actually, with data does takes a parameter. It's not always applicable though...
But try experiment with `&format=atom&with-data=(trig|nquads|json|etc'...)`

In any case, if it'll be refactored down the road with a new type, it makes even more sense to write it in as few places as possible... Anyway, it doesn't matter that much...